src/Microsoft.ML.StaticPipe/TextStaticExtensions.cs

```diff
@@ -492,15 +492,15 @@ public bool Equals(Reconciler other)
         /// <param name="ngramLength">Ngram length.</param>
         /// <param name="skipLength">Maximum number of tokens to skip when constructing an ngram.</param>
         /// <param name="allLengths">Whether to include all ngram lengths up to <paramref name="ngramLength"/> or only <paramref name="ngramLength"/>.</param>
-        /// <param name="maxNumTerms">Maximum number of ngrams to store in the dictionary.</param>
```

Comment:
I think we trying to move away from `Terms`. Ngrams is more appropriate

Reply:
They are different things. Terms are tokens while n-grams are combinations of tokens.